kalite/i18n/base.py

```diff
@@ -89,7 +89,7 @@ def get_srt_path(lang_code=None, youtube_id=None):
     if lang_code:
         srt_path = os.path.join(srt_path, lcode_to_django_dir(lang_code), "subtitles")
     if youtube_id:
-        srt_path = os.path.join(srt_path, youtube_id + ".srt")
+        srt_path = os.path.join(srt_path, youtube_id + ".vtt")
```

Comment:
Not so certain about these changes. Somehow this slipped by me in previous PRs, but it's used for instance to build correct paths/urls in tests. Should I create new functions, instead of modifying these @aronasorman?